samza-core/src/main/scala/org/apache/samza/util/Util.scala

```diff
@@ -62,6 +62,21 @@ object Util extends Logging {
     }
   }
 
+  /**
+    * Instantiate an object from given className, and given constructor parameters.
+    */
+  def getObj(className: String, constructorParams: (Class[_], Object)*) : Any = {
```

Comment:
Should not return Any, can take [T] as a generic param and require passing the type (see other variant)